Database/Corrections/sodQuestFixes.lua

```diff
@@ -2358,6 +2358,66 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -424992,
             [questKeys.zoneOrSort] = sortKeys.ROGUE,
         },
+        [90139] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {{1161,1162,1163,1164}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 14,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.WARRIOR,
+            [questKeys.objectivesText] = {"Kill Troggs for the Skull-Shaped Geode. Then get hit by a Skullthumper's stun and use the geode to loot the rune."},
+            [questKeys.requiredSpell] = -425443,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
+        },
+        [90140] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {{504,95,449,590,589,121,285,46,732,735},{408799}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 20,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.WARRIOR,
+            [questKeys.objectivesText] = {"Kill Defias for Gillsbane, equip it and then kill Murlocs to receive Soul of the Sea. Take it to the Idol of the Deep to summon Horror of the Deep and defeat it."},
+            [questKeys.requiredSpell] = -425443,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
+        },
+        [90141] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {nil,{408802}},
+            [questKeys.finishedBy] = {210482},
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 20,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.WARRIOR,
+            [questKeys.objectivesText] = {"Loot the Gnarled Harpoon, and use it on Paxnozz to weaken it, kill it and loot the rune."},
+            [questKeys.requiredSpell] = -425443,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
+        },
+        [90142] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {{3497}},
+            [questKeys.finishedBy] = {209797},
```

Comment:
```suggestion
            [questKeys.startedBy] = {{3497,209797}},
            [questKeys.finishedBy] = nil,
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5435/commits/d88896bc591f70a14712d169d02ab3467c35096a